src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -0,0 +1,47 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.Cart;
+import com.kodilla.ecommercee.domain.CartDto;
+import com.kodilla.ecommercee.mapper.CartMapper;
+import com.kodilla.ecommercee.service.CartService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Objects;
+
+@RestController
+@RequestMapping("v1/cart")
+public class CartController {
+    @Autowired
+    private CartService cartService;
+
+    @Autowired
+    private CartMapper cartMapper;
+
+    @GetMapping
+    public Cart createEmptyCart(){
+        return new Cart();
+    }
+
+    @GetMapping(value="getById")
+    public CartDto getCartById(@RequestParam Long cartId){
+        return cartMapper.mapToCartDto(Objects.requireNonNull(cartService.findById(cartId).orElse(null)));
```

Comment:
mega długa linijka, przez co nie jest czytelna... nie uważasz, że można to podzielić na kilka linijek co znacząco zwiększy czytelność?